auth-demo/README.md

```diff
@@ -0,0 +1,28 @@
+# auth-demo
+
+## QR code integration
+
+![login page](login.png)
+
+The QR code encodes JSON containing the challenge DID and the callback URL, e.g.
```

Comment:
We're trying to save characters, because QR codes are limited in how much data they can contain. But for a process like auth, obfuscating what parameters mean a little bit is probably not the worst thing.